examples/data-sources/nios_ipam_ipv6networktemplate/data-source.tf

```
@@ -0,0 +1,16 @@
+// Retrieve a specific IPAM IPv6 Network Template using filters
+data "nios_ipam_ipv6networktemplate" "get_ipam_ipv6network_template_using_filters" {
+  filters = {
+    name = "example_ipv6_network_template"
```

```suggestion
    name = "example_network_template"
```
As this name is used in resource examples